### ISSUE TITLE
Links are not clearly identifiable

### ISSUE THREAD

**Issue by poojagunturu96** [middlebury/midd-frontend#601]
Color alone is not enough to distinguish a link from the surrounding text. Providing another visual cue —such as underlining or bolding the link text— will also make it easier for non-colorblind people (especially people with low vision) to scan the page.

Issue occurs mainly in tags for news stories:
<img width="671" alt="Screenshot 2024-03-18 at 4 51 21 PM" src="https://github.com/middlebury/midd-frontend/assets/79096552/b4e1ac06-47c6-4b3d-9208-878ce8730fbd">

Siteimprove report for this issue: https://my2.siteimprove.com/Accessibility/7333/NextGen/Issue/1?ruleId=62&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2

**Comment by poojagunturu96** [comment]
Looking at [Failure of Success Criterion 1.4.1](https://www.w3.org/TR/2008/WD-WCAG20-TECHS-20081103/F73), it mentions:

> **While some links may be visually evident from page design and context**, such as navigational links, links within text are often visually understood only from their own display attributes.

This implies that not having an underline on the links highlighted in the screenshot above is OK as the page design and context make it evident that those are links.
